lib/manager/selectors/__tests__/index.js

```diff
@@ -157,6 +159,7 @@ describe('manager > selectors >', () => {
             ],
             trips: [
               {
+                frequencies: [],
```

Comment:
Would be nice to either modify this test case or have another one for when frequencies do exist.